src/ModelerApp.vue

```diff
@@ -69,16 +78,17 @@ export default {
     `;
 
     this.$refs.modeler.loadXML(blank);
+    for(var node of initialNodes) {
+      this.$refs.modeler.registerNode(node)
```

Comment:
Instead of reaching into the child through a ref, why not handle this in the `Modeler` component directly?

Reply:
The reason being that the modeler component is embedded into bpm4.  In that situation, different nodes can be registered instead of the ones that are part of the package.  This is how overriding the inspector configs for certain nodes when the modeler is used in bpm4 is done.